<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAssembly</class>
 <widget class="QDialog" name="DialogAssembly">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Control Override</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Max Torque</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Max Force</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="LineEditDouble" name="lineEditMaxForce"/>
      </item>
      <item row="2" column="1">
       <widget class="LineEditDouble" name="lineEditAngleGain"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Position Gain</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="LineEditDouble" name="lineEditMaxTorque"/>
      </item>
      <item row="3" column="1">
       <widget class="LineEditDouble" name="lineEditPositionGain"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Angle Gain</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBoxBodyPose">
     <property name="title">
      <string>Body Pose</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Body</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QComboBox" name="comboBoxBodyList"/>
      </item>
      <item row="1" column="1">
       <widget class="LineEditDouble" name="lineEditX"/>
      </item>
      <item row="1" column="2">
       <widget class="LineEditDouble" name="lineEditY"/>
      </item>
      <item row="1" column="3">
       <widget class="LineEditDouble" name="lineEditZ"/>
      </item>
      <item row="2" column="1">
       <widget class="LineEditDouble" name="lineEditXR"/>
      </item>
      <item row="2" column="2">
       <widget class="LineEditDouble" name="lineEditYR"/>
      </item>
      <item row="2" column="3">
       <widget class="LineEditDouble" name="lineEditZR"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Position (XYZ)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Euler (XYZ)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBoxJoints">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
     <property name="title">
      <string>Joints</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item row="0" column="3">
       <widget class="QPushButton" name="pushButtonCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButtonOK">
        <property name="text">
         <string>OK</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>454</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButtonReset">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEditDouble</class>
   <extends>QLineEdit</extends>
   <header>LineEditDouble.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEditMaxTorque</tabstop>
  <tabstop>lineEditMaxForce</tabstop>
  <tabstop>lineEditAngleGain</tabstop>
  <tabstop>lineEditPositionGain</tabstop>
  <tabstop>comboBoxBodyList</tabstop>
  <tabstop>lineEditX</tabstop>
  <tabstop>lineEditY</tabstop>
  <tabstop>lineEditZ</tabstop>
  <tabstop>lineEditXR</tabstop>
  <tabstop>lineEditYR</tabstop>
  <tabstop>lineEditZR</tabstop>
  <tabstop>pushButtonOK</tabstop>
  <tabstop>pushButtonCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
